util/opds_writer.py

```diff
@@ -47,7 +48,8 @@ class AtomFeed(object):
         'schema' : SCHEMA_NS,
         'simplified' : SIMPLIFIED_NS,
         'bibframe' : BIBFRAME_NS,
-        'bib': BIB_SCHEMA_NS
+        'bib': BIB_SCHEMA_NS,
+        'opensearch': OPENSEARCH_NS,
```

Comment:
This is a tiny core change for https://jira.nypl.org/browse/SIMPLY-2422 so I don't have to do a whole other core branch. It's just making sure we can use tag names from the OpenSearch namespace in our OPDS documents.